src/assets/scripts/client/base/ModelSource/ModelSourcePool.js

```diff
@@ -91,9 +91,13 @@ class ModelSourcePool extends BaseCollection {
     releaseReusable(constructorName, ...args) {
         let model = this._findModelByConstructorName(constructorName);
 
-        if (!model) {
-            model = new CLASS_MAP[constructorName](...args);
+        if(model){
+            model.init(...args);
+            return model;
```

Comment:
I'd like to see the logic reversed here.  The code inside the if block should happen only when `!model`.  pay close attention to spacing and line breaks too.  There should be a space after the word `if` and an empty line above `return`.